### ISSUE TITLE
LogERROR in download/download.py:171

### ISSUE THREAD

**Issue by lots0logs** [Antergos/Cnchi#448]
## Exception in Cnchi

**LogERROR** in **download/download.py:171**
Error creating metalink for package gnome-shell-extension-status-menu-buttons. Installation will stop

[View on Bugsnag](https://bugsnag.com/antergos/cnchi/errors/55edf7620f76499ec466fa76?event_id=5618ee2ecd4bcf563b23c9a6)

## Stacktrace

    download/download.py:171 - create_metalinks_list
    misc/misc.py:175 - helper
    installation/process.py:79 - run
    summary.py:172 - store_values
    main_window.py:401 - on_forward_button_clicked
    cnchi.py:484 - &lt;module&gt;

[View full stacktrace](https://bugsnag.com/antergos/cnchi/errors/55edf7620f76499ec466fa76?event_id=5618ee2ecd4bcf563b23c9a6)

**Comment by lots0logs** [comment]
@karasu This issue occurs when a package has been recently updated in the main repo and all the mirrors have not synced yet. Though the change I made to the metalinks should have resolved this for the most part. I think we need to ensure that our main repo is NOT used in dependency calculation (that way if a package was recently updated on the main repo we won't try to get the new version that hasn't been synced to the other mirrors yet). Thoughts??

**Comment by karasu** [comment]
> I think we need to ensure that our main repo is NOT used in dependency calculation

Yes, I think this is the answer. Do you know (more or less) how much time do the servers need to update themselves after a main repo update? 24 hours? More?

**Comment by lots0logs** [comment]
The mirror-brain server syncs with the build server every hour. Tier 1 mirrors sync every hour with the mirror-brain server. Tier 2 mirrors sync every hour from a tier 1 mirror. So, basically, the longest delay should be only 2 hours (provided all the mirrors are syncing on schedule).
